MobileCenter/MobileCenter/Internals/Sender/MSSender.h

```diff
@@ -11,7 +11,17 @@ NS_ASSUME_NONNULL_BEGIN
 @protocol MSSender <NSObject, MSSenderCallDelegate, MSEnable>
 
 /**
- *  Reachability library.
+ * Base URL (schema + authority + port only) used to communicate with the server.
```

Comment:
This is a generic sender. I don't think we need to place those properties in here. Need to go to http sender, what do you think?